walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

```diff
@@ -71,23 +71,23 @@ The Fuse Aggregation App provides information about flights. You deploy it using
 
 . Select the option to *Build and Deploy Online* in the *Target Environment* step.
 
-. Select the *Authorize* button to allow Launcher create resources in the OpenShift cluster. This triggers an OAuth flow, and eventually redirects back to the {launcher-service} wizard.
+. Click the *Authorize* button to allow Launcher create resources in the OpenShift cluster. This triggers an OAuth flow, and eventually redirects back to the {launcher-service} wizard.
 
-. Select the blue down-arrow icon to continue the wizard.
+. Click the blue down-arrow icon to continue the wizard.
```

Comment:
I think it would read better if there was some sort of styling applied to "blue down-arrow". When I look at the next sentence "*Aggregation* mission", it makes sense to me to continue with this styling.

Reply:
Will put the "blue" in italics.